components/Transaction/index.tsx

```diff
@@ -127,10 +136,18 @@ const Transaction = ({
         </View>
       </View>
       <View className="flex-row items-center gap-2 md:gap-4 flex-shrink-0">
-        <Text className={cn('text-lg font-medium text-right', statusTextColor)}>
-          {statusSign}
-          {formatNumber(Number(amount))} {symbol?.toLowerCase() === 'sousd' ? 'soUSD' : symbol}
-        </Text>
+        {shouldShowWaitingMessage ? (
+          <Text className="text-sm text-muted-foreground font-medium text-right">
+            Waiting for transfer
+          </Text>
+        ) : shouldShowFailedMessage ? (
```

Comment:
@SametSahin10 remove shouldShowFailedMessage. If transaction is failed, it's already indicated through FAILED statusSign and red statusTextColor
https://github.com/Solid-Money/solid-ui/blob/a323f3d83854fee03b1fcdfbdb468323a2f0898e/lib/types.ts#L339